lib/controllers/links.js

```diff
@@ -0,0 +1,29 @@
+const { Router } = require('express');
+const Link = require('../models/Link');
+
+module.exports = Router()
+  .post('/', (req, res, next) => {
+    Link
+      .insert(req.body)
+      .then(link => res.send(link))
+      .catch(next);
+  })
+  .get('/:id', (req, res, next) => {
+    Link
+      .findById(req.params.id)
+      .then(link => res.send(link))
+      .catch(next);
+  })
+  .get('/', (req, res, next) => {
+    Link
+      .find()
+      .then(link => res.send(link))
+      .catch(next);
+  })
+  .delete('/:id', (req, res, next) => {
+    Link
+      .delete(req.params.id)
+      .then(link => res.send(link))
+      .catch(next);
+  });
+
```

Comment:
Are we missing a put/UPDATE route?

Reply:
I didn't think it would be necessary since these are just links. I couldn't think of a scenario where we would need to "edit" a link vs either adding or deleting.